content/en/about/_index.md

```diff
@@ -20,7 +20,7 @@ aliases:
 
   ![What is runtime security and why it matters](/img/about/runtime_security.svg#img-fit)
 
-  To effectively detect and respond to threats, runtime security solutions must be efficient and alert on suspicious behavior in real time. Attackers have adapted their tactics to the landscape of the cloud and can initiate attacks within seconds of entering your environment. While legacy solutions struggle to detect cloud-native threats, Falco can provide a layer of protection to identify and respond to these incidents as they happen. This will minimize the impact of breaches and protect your organization against financial loss and reputational damage.
+  To effectively detect threats, runtime security solutions must be efficient and alert on suspicious behavior in real time. Attackers have adapted their tactics to the landscape of the cloud and can initiate attacks within seconds of entering your environment. While legacy solutions struggle to detect cloud-native threats, Falco can provide a layer of protection to identify these incidents as they happen. This will give you an opportunity to take the right response action, and minimize the impact of breaches and protect your organization against financial loss and reputational damage.
```

Comment:
proposing a small rewrite as it seems better to stay with terms like "abnormal behavior" as at that point it is not yet know if it is a security incident, hence would stay clear from incident and response terms for the most part, but with a small rewrite we can still keep response actions, it fits in well here.

And definitely Falco is not exactly protecting, while I get what is meant, we could avoid using that term as it is often used in the context of blocking actions etc and this is why confusion may arise?

` ... While legacy solutions struggle to detect cloud-native threats, Falco can provide a sophisticated security monitoring layer to identify abnormal behaviors as they happen. This will give you an opportunity to take the right response action and minimize the impact of possible breaches, including sustained financial loss and reputational damage.`